src/ptvsd/common/attach_script_ptvsd_pid.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Since it's already inside the `ptvsd` package, the name seems redundant. Maybe something like `attach_pid_injected.py`?

Also, I don't think this should be under `ptvsd.common` - it's the implementation detail of attach-by-PID code that is inside the server specifically, and nothing else can use it for any other purpose, so it should be in `ptvsd.server` as well.